README.md

```diff
@@ -341,10 +341,10 @@ echo (string) $ipv6;
 echo $ipv6->toIPv4();
 
 // IPv4-mapped notation
-$ipv6 = $ipv4->toIPv6IPv4Mapped();
+$ipv6_6to4 = $ipv4->toIPv6IPv4Mapped();
```

Comment:
The result of ` $ipv4->toIPv6IPv4Mapped()` is an instance of `IPLib\Address\IPv6`, isn't it?

So, why is this a typo?